src/View/mainpage/DicomView.py

```diff
@@ -155,7 +155,7 @@ def isodose_display(self):
             # sort selected_doses in ascending order so that the high dose isodose washes
             # paint over the lower dose isodose washes
             for sd in sorted(self.patient_dict_container.get("selected_doses")):
-                dose_level = sd * self.patient_dict_container.get("rxdose") / \
+                dose_level = sd * self.patient_dict_container.get("rx_dose_in_cgray") / \
                              (dataset_rtdose.DoseGridScaling * 10000)
```

Comment:
10000 is a magic number.
What does it represent?  What are it's units (or is it unitless) ?